src/components/FiltersCollection.jsx

```diff
@@ -1,12 +1,43 @@
 import '../styles/search.css'
 import React, { useState } from 'react'
-import { applySearchNameAsync, getAllItemsAsync } from '../redux/items/thunks'
+import { applySearchNameAsync, getAllItemsAsync, applyFiltersAsync } from '../redux/items/thunks'
 import { useDispatch } from 'react-redux'
 import { Button, ButtonToolbar, InputGroup, Form } from 'react-bootstrap'
+import { ITEM_TYPES, RATINGS } from '../constants'
 
 function FiltersCollection () {
   const dispatch = useDispatch()
   const [searchInput, setSearchInput] = useState('')
+  const [itemType, setItemType] = useState('Select item type...')
+  const [itemRate, setItemRate] = useState('Unrated')
```

Comment:
You don't need to set the default message here. You can just add a default option like you have in the item type dropdown.

Reply:
But the problem with that is after you click the second dropdown, it goes back to the default option.

Reply:
> But the problem with that is after you click the second dropdown, the current dropdown goes back to the default option.

Sorry not sure what you mean. If I change it to this everything works as expected: 

```
const [itemType, setItemType] = useState('')
const [itemRating, setItemRating] = useState('')
...
<Form.Group >
    <Form.Select value={itemType} onChange={handleCategory}>
      <option>Select item type...</option>
      {itemTypeDropdowns}
    </Form.Select>
  </Form.Group>
<Form.Group >
  <Form.Select value={itemRating} onChange={handleRating}>
    <option>Select a rating...</option>
    {ratingDropdowns}
  </Form.Select>
</Form.Group>
```